text/0000-attribute-actions.md

```diff
@@ -0,0 +1,244 @@
+- Start Date: 2015-10-17
+- RFC PR: https://github.com/emberjs/rfcs/pull/100
+- Ember Issue: (leave this empty)
+
+# Summary
+
+Ember's `action` helper has two inconsistent forms. The goal of this RFC is to
+allow the deprecation or removal of the classic action helper by adding a new
+DOM event listener API designed for closure actions.
+
+The RFC suggests three phases of implementation:
+
+* In Ember 2.x (after this RFC is merged), attribute actions like
+  `<button onclick={{action 'save'}}>` are declared an
+  official API.
+  * There are some very small code changes, but largely this is a
+    messaging and documentation change.
+  * These "attribute events" should use `element.addEventListener(eventName, handler, false)`
+    to attach instead of `element['on'+eventName'] = handler` for attachment.
+  * Non-action functions should warn if they are passed to `onclick=`
+    APIs.
+  * Non-functions should not be permitted as a bound value for `onclick=`, for
```

Comment:
`on*`. The list of words starting with `on` is fairly forgiving:

* http://www.wordfind.com/starts-with/on/

Open to feedback, since this is pre-glimmer components there should be few `on*` attributes being passed around that were not already intended as as event listeners.